reana_job_controller/config.py

```diff
@@ -90,5 +90,19 @@
                                 f'{REANA_COMPONENT_PREFIX}-krb5-conf')
 """Kerberos configMap name."""
 
+PROXY_CONTAINER_IMAGE = os.getenv('PROXY_CONTAINER_IMAGE',
```

Comment:
What do you think about using the  `AUTH_[VOMS_PROXY|KRB5]` prefix, following the same name convention as the Docker images?

Reply:
You're right, best to stick to same name convention as much as possible!